src/app/_components/CardLayout.tsx

```diff
@@ -3,7 +3,7 @@ import clsx from 'clsx'
 type CardLayoutProps = {
   children: React.ReactNode
   as?: React.ElementType
-  type?: 'default' | 'blogPost' | 'caseStudy' | 'reports'
+  type?: 'default' | 'blogPost' | 'caseStudy' | 'reports' | 'grants'
```

Comment:
Let's alphabetise after `default`

```suggestion
  type?: 'default' | 'blogPost' | 'caseStudy' | 'grants' | 'reports' 
```